geosys/bridge_api/test/test_connection.py

```diff
@@ -32,8 +32,8 @@ def tearDown(self):
     def test_access_token(self):
         """Test we can successfully get the access token."""
         client = ConnectionAPIClientV2(IDENTITY_URLS['na']['test'])
-        json = client.get_access_token('Bridge_US_Demo', 'Welcome12k18')
-        self.assertTrue('access_token' in json)
+        response = client.get_access_token('Bridge_US_Demo', 'Welcome12k18')
```

Comment:
Ummm....I don't know if we can do this Rohmat - we should manage these as environment variables and explain in the README how to set things up for local testing and for travis. Otherwise you are exposing their password for all who view the code..

Reply:
of course! my bad, I will at least make it not hardcoded in the code